contracts/AuraBribesProcessor.sol

```diff
@@ -278,12 +208,17 @@ contract AuraBribesProcessor is CowSwapSeller {
 
         // Sends Badger to the Tree
         // Emits custom event for it
-        uint256 toEmitTotal = BADGER.balanceOf(address(this));
-        require(toEmitTotal > 0);
+        uint256 totalBadger = BADGER.balanceOf(address(this));
+        require(totalBadger > 0);
+
+        uint256 ops_fee = totalBadger * OPS_FEE / MAX_BPS;
```

Comment:
@GalloDaSballo, the issue I was referring to in the call is that if BADGER is claimed as a bribe, it will be directly emitted to the Tree from the strategy: https://etherscan.io/address/0x0d724e8aee6f73b35a596c8c947c92c75eac7818#code#F1#L2075

This is accounted in the following [script](https://github.com/Badger-Finance/badger-multisig/pull/692) in order to get a 25/75 split whenever possible but it doesn't fix the fact that the BADGER acquired from wETH and processed through this contract may not be the total amount of BADGER obtained from Bribes. Hence, this fee will only be processed on the portion of BADGER acquired from wETH. This can be significant because, for instance, next round we are mainly getting BADGER.

Reply:
One way to approach this would be to charge a variable fee (higher than 5%) on this portion of BADGER to account to the already emitted amount. The issues with this is that it could introduce a rug vector since there wouldn't be a way to cap it as all of the BADGER being processed may be 5% of the total BADGER from bribes. Another issue is that there's a high chance that more than 25% of bribes will be in the form of BADGER, in these scenarios we won't be buying any more and all of the BADGER claimed will be directly emitted without fees.

Charging it on the wETH is not very viable either as long as BADGER and AURA bribes are received directly since the wETH will not represent the full amount of bribes.

Only thing I can think of is to somehow be able to charge a variable fee that is estimated off-chain via scripts but I can't think of how could it be properly capped - plus, it would be a step away from the autonomous and trustless goal for the processor. Will give it some more though.

Maybe @gosuto-inzasheru and @petrovska-petro can weight in in here.

Reply:
we discussed after the call this same topic with @gosuto-inzasheru and @GalloDaSballo re the badger which will be redirected directly to tree via strat. Consensus was:

- track the amounts being transferred over

- currently the take is not worth the update just for the inclusion of the ops fee over the **_sendBadgerToTree**

- if amounts starts to be outrageous, start evaluating the upgrade. but given the bribes given by badger comes from our end, unlikely to see massive amounts being bribed. currently around 10k

- we can track via the autovoter which % of the vote is assigned to this gauge and we will have reports moving forward per round

Reply:
Sounds good to me. We will continue to keep an eye on the opportunity to cost of those missing fees, if it turns out to be too big, we will upgrade the strategy to handle.

Thanks!